gradle/libs.versions.toml

```diff
@@ -32,7 +36,7 @@ composeCompiler = "1.3.0"
 composeDestinations = "1.6.15-beta"
 composeFoundation = "1.2.1"
 composeMaterial = "1.2.1"
-composeMaterial3 = "1.0.0-alpha16"
+composeMaterial3 = "1.0.0-beta03"
```

Comment:
this is RC-01 now https://developer.android.com/jetpack/androidx/releases/compose-material3#1.0.0-rc01

Reply:
It's not important for this PR, I'll update this and a couple other libraries in a consequent PR.